### DISCUSSION TITLE
How Can I Trigger a GitHub Action Only on Specific Pull Request Labels?

### DISCUSSION THREAD

**Discussion by 48Naveenkumar** [community/community#138177] [GitHub Education] [answered]
### Hi everyone,

Is there a way to configure a GitHub Action to run only when a pull request has a specific label applied, rather than on every PR event? I'm trying to limit the workflow runs to specific scenarios based on labels. Any guidance on how to set this up?

**Comment by 48Naveenkumar** [comment, marked as answer]
To trigger a GitHub Action based on specific pull request labels, you can use a combination of types: [labeled] in the workflow and a conditional if statement to check the label name.

**Reply by tennengabr** [reply]
This leads a Skipped CI being displayed in a PR. Which is less than ideal.
